.github/workflows/tos.yaml

```diff
@@ -144,6 +148,9 @@ jobs:
         run: exit 1
 
   validate:
+    concurrency:
+      group: submission-${{ github.event.issue.number }}
+      cancel-in-progress: true
```

Comment:
Should this be moved at the workflow level (instead of job level) inside the `./.github/workflows/validation.yaml` file?

Reply:
If it is at the workflow level, whenever there is a comment that is not `/validate` the run will be cancelled because this workflow listens to `issue_comment/created` event. 
Moving it to the workflow level requires more rework.